e2e/ansible/playbooks/hyperconverged/test-k8s-memleak/memleak-vars.yml

```diff
@@ -2,7 +2,8 @@
 test_name: volume_memcheck_test
 volume_def: test-pvc.yaml
 io_duration: 200s
-block_size: 32k
+block_size: 512k
+block_size1: 1m
```

Comment:
Can you create a list with various block sizes - you can refer in the playbook via index